validphys2/src/validphys/tests/photon/test_compute.py

```diff
@@ -25,62 +29,182 @@ def generate_fiatlux_runcard():
     }
 
 
-def test_parameters_init():
-    "test initailization of the parameters from Photon class"
-    fiatlux_runcard = generate_fiatlux_runcard()
-    test_theory = API.theoryid(theoryid=THEORY_QED)
+# def test_parameters_init():
+#     """test initailization of the parameters from Photon class"""
+#     fiatlux_runcard = generate_fiatlux_runcard()
+#     test_theory = API.theoryid(theoryid=THEORY_QED)
 
-    # we are not testing the photon here so we make it faster
-    fiatlux_runcard['eps_base'] = 1e-1
+#     # we are not testing the photon here so we make it faster
+#     fiatlux_runcard['eps_base'] = 1e-1
 
-    photon = Photon(test_theory, fiatlux_runcard, [1, 2, 3])
+#     photon = Photon(test_theory, fiatlux_runcard, [1, 2, 3])
 
-    np.testing.assert_equal(photon.replicas, [1, 2, 3])
-    np.testing.assert_equal(photon.luxpdfset._name, fiatlux_runcard["luxset"].name)
-    np.testing.assert_equal(photon.additional_errors.name, "LUXqed17_plus_PDF4LHC15_nnlo_100")
-    np.testing.assert_equal(photon.luxseed, fiatlux_runcard["luxseed"])
-    np.testing.assert_equal(photon.path_to_eko_photon, test_theory.path / "eko_photon.tar")
-    np.testing.assert_equal(photon.q_in, 100.0)
+#     np.testing.assert_equal(photon.replicas, [1, 2, 3])
+#     np.testing.assert_equal(photon.luxpdfset._name, fiatlux_runcard["luxset"].name)
+#     np.testing.assert_equal(photon.additional_errors.name, "LUXqed17_plus_PDF4LHC15_nnlo_100")
+#     np.testing.assert_equal(photon.luxseed, fiatlux_runcard["luxseed"])
+#     np.testing.assert_equal(photon.path_to_eko_photon, test_theory.path / "eko_photon.tar")
+#     np.testing.assert_equal(photon.q_in, 100.0)
```

Comment:
Please remove all these commented lines.

Reply:
All the `#` were accidentally pushed :)
Fixed it